.github/workflows/test.yml

```diff
@@ -79,7 +79,17 @@ jobs:
           cache-provider: "github"
       - name: Check that the lockfile is updated
         run: cargo fetch --locked
-    
+
+  audit:
+    runs-on: buildjet-2vcpu-ubuntu-2204
```

Comment:
we can probably run this on a standard ubuntu runner. We use the buildjet for things that require more compute power, but can use standard github runners in many places.